src/main/kotlin/at/bitfire/dav4jvm/property/webdav/GetLastModified.kt

```diff
@@ -24,22 +22,15 @@ data class GetLastModified(
     }
 
 
-    object Factory: PropertyFactory {
+    object Factory: PropertyFactory<GetLastModified> {
 
         override fun getName() = NAME
 
-        override fun create(parser: XmlPullParser): GetLastModified? {
+        override fun create(parser: XmlPullParser): GetLastModified {
             // <!ELEMENT getlastmodified (#PCDATA) >
-            XmlUtils.readText(parser)?.let { rawDate ->
-                val date = HttpUtils.parseDate(rawDate)
-                if (date != null)
-                    return GetLastModified(date)
-                else {
-                    val logger = Logger.getLogger(javaClass.name)
-                    logger.warning("Couldn't parse Last-Modified date")
-                }
-            }
-            return null
+            return GetLastModified(
+                readInstantOrNull(parser)
```

Comment:
Fine :)

Maybe still name it `readHttpDateOrNull`, because it expects an HTTP date, while an `Instant` could be an some other representation (for instance a timestamp).